From 63d72f97bd106dd2101cd7fdac6df4f7a053d67c Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 2 Sep 2022 08:27:39 -0700
Subject: [PATCH] configure: Remove the logic to download IANA PEN database
 during configure

OE will do all downloading before it starts to configure therefore this
step is moved out into bitbake recipe, so we can make it immutable build

Upstream-Status: Inappropriate [OE-Specific]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 configure.ac | 16 +---------------
 1 file changed, 1 insertion(+), 15 deletions(-)

--- a/configure.ac
+++ b/configure.ac
@@ -56,21 +56,7 @@ if test "x$exec_prefix" = "xNONE"; then
 	exec_prefix="$prefix"
 fi
 
-if test "x$WGET" = "x"; then
-	if test "x$CURL" = "x"; then
-		AC_MSG_WARN([** Neither wget nor curl could be found.])
-		AC_MSG_WARN([** IANA PEN database will not be installed by `make install` !])
-	else
-		DOWNLOAD="$CURL --location --progress-bar"
-		AM_CONDITIONAL([DOWNLOAD], [true])
-	fi
-else
-	DOWNLOAD="$WGET -c -nd -O -"
-	AM_CONDITIONAL([DOWNLOAD], [true])
-fi
-
-AC_MSG_WARN([** Download is:])
-AC_MSG_WARN($DOWNLOAD)
+AM_CONDITIONAL([DOWNLOAD], [false])
 AC_SUBST(DOWNLOAD, $DOWNLOAD)
 
 dnl
